packages/api/controllers/grantCycle.js

```diff
@@ -48,8 +48,25 @@ const update = async (req, res) => {
 
 const findAll = async (req, res) => {
   try {
-    const grants = await db.GrantCycle.findAll();
+    const grants = await db.GrantCycle.findAll({ order: [['openedOn', 'DESC']], raw: true });
     if (grants.length) {
+      const result = await Promise.all(grants.map(async (g) => {
+        try {
+          const nominations = await db.Nomination.findAll({ 
+            where: {
+              dateReceived: {
+                [Op.between]: [g.openedOn, g.closedOn]
+              }
+            }
+          , order: [
+            ['dateReceived', 'DESC' ]
+          ]});
+          g.nominations = nominations;
+        }
+        catch (error) {
+          return res.status(404).send(error);
+        }
+      }));
```

Comment:
Finds all grant cycles and also add an array of all the nominations where the date received falls within the grant cycle dates.